destroy_cluster.sh

```diff
@@ -10,8 +10,13 @@ else
     echo "Please export CONFIG. See readme for help."
     exit 1
 fi
-CLUSTER_PREFIX=$CLUSTER_NAME
-CLUSTER_NAME=$(jq .infraID $CLUSTER_PREFIX/metadata.json | sed "s/\"//g")
+
+HASJQ="$(which jq)"
+if [ -n $HASJQ ] && [ -f $CLUSTER_NAME/metadata.json ]; then
+    CLUSTER_PREFIX=$CLUSTER_NAME
+    CLUSTER_NAME=$(jq .infraID $CLUSTER_PREFIX/metadata.json | sed "s/\"//g")
```

Comment:
Since you're re-using the CLUSTER_NAME variable, you don't need to define a new CLUSTER_PREFIX variable that isn't used anywhere else. It's a bit confusing.

You could simplify to:
`CLUSTER_NAME=$(jq .infraID $CLUSTER_NAME/metadata.json | sed "s/\"//g")`

Perhaps also add a comment explaining why in certain cases you need a new value for CLUSTER_NAME.